db/psychologists.js

```diff
@@ -103,14 +118,17 @@ module.exports.savePsychologistInPG = async function savePsychologistInPG(psyLis
   const updatedAt = date.getDateNowPG(); // use to perform UPSERT in PG
   const universities = await dbUniversities.getUniversities();
 
+  const psychologists = await getPsychologistsByIds(psyList.map((psy) => psy.dossierNumber));
+  const psychologistsToInsert = [];
   const upsertArray = psyList.map(async (psy) => {
     psy.languages = addFrenchLanguageIfMissing(psy.languages);
     psy.assignedUniversityId = dbUniversities.getAssignedUniversityId(psy, universities);
```

Comment:
This can be done only if `!psyInDB` since we never update this field in other cases.